src/UKHO.S100PermitService.StubService/Stubs/HoldingsServiceStub.cs

```diff
@@ -1,13 +1,21 @@
 ﻿using System.Net;
 using UKHO.S100PermitService.StubService.Configuration;
+using WireMock;
+using WireMock.Matchers;
 using WireMock.RequestBuilders;
 using WireMock.ResponseBuilders;
 using WireMock.Server;
+using WireMock.Types;
+using WireMock.Util;
 
 namespace UKHO.S100PermitService.StubService.Stubs
 {
     public class HoldingsServiceStub : IStub
     {
+        private const string APPLICATIONTYPE = "application/json";
+        private const string RESPONSEFILEDIRECTORY = "StubData\\Holdings";
+        private readonly string RESPONSEFILEDIRECTORYPATH = Path.Combine(Environment.CurrentDirectory, RESPONSEFILEDIRECTORY);
```

Comment:
Should be 'ResponseFileDirectoryPath', it's not const

Reply:
Also, can you please make const strings Pascal case, not capitalized? Thanks!

Reply:
Changed const to Pascal case